lib/pbio/include/pbsys/program_load.h

```diff
@@ -41,6 +41,13 @@ typedef struct _pbsys_program_load_data_header_t {
      */
     volatile uint32_t checksum_complement;
     #endif
+    /**
```

Comment:
I'm not sure if this is correct. It looks like one of the drivers requires a granularity of 64 bits.